buildscripts/condarecipe_clone_icc_rt/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set version = "2018.0.2" %} # this is the intel version to get
+{% set build_number = "0" %}   # and the build number from the intel version
+
+package:
+  name: icc_rt
+  version: {{ version }}
+
+build:
+  number: {{ build_number }}
+
+source:
+    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/win-64/icc_rt-{{version}}-intel_{{build_number}}.tar.bz2 # [win]
+    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/osx-64/icc_rt-{{version}}-intel_{{build_number}}.tar.bz2 # [osx]
+    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/linux-64/icc_rt-{{version}}-intel_{{build_number}}.tar.bz2 # [linux]
```

Comment:
I know it is pedantic, but presumably we need to put a SHA256 for each of these?

Reply:
I think we can, or if we don't it just downloads them fresh. Does SHA work with selectors (am going to assume so).

Reply:
yes, selectors should work with any of the config lines, I think

Reply:
done in https://github.com/numba/numba/pull/2881/commits/976fcd61ff191523c471f9535db05c24b882fc6a